pynsee/localdata/get_area_list.py

```diff
@@ -106,16 +110,17 @@ def get_area_list(area=None, update=False):
             inplace=True,
         )
         data_all.to_pickle(file_data)
-        print(f"Data saved: {file_data}")
+        logger.debug(f"Data saved: {file_data}")
     else:
         try:
             data_all = pd.read_pickle(file_data)
         except:
             os.remove(file_data)
             data_all = get_area_list(area=area, update=True)
         else:
-            print(
-                f"Locally saved data has been used\nSet update=True to trigger an update"
+            logger.info(

```

Comment:
discuss visibility (info vs warning) cc @hadrilec

Reply:
Same as previous, relates to documentation about update.